CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### 🛑 Breaking changes 🛑
 
+### 🚩 Deprecations 🚩
+
+### 💡 Enhancements 💡
+
+### 🧰 Bug fixes 🧰
+
+## v0.51.0 Beta
+
+### 🛑 Breaking changes 🛑
+
 - Remove deprecated model module, everything is available in `pdata` and `semconv`. (#5281)
   - Old versions of the module are still available, but no new versions will be released.
 - Remove deprecated LogRecord.Name field. (#5202)
 
-### 🚩 Deprecations 🚩
-
 ### 💡 Enhancements 💡
 
 - `pdata`: Expose `pcommon.NewSliceFromRay` and `pcommon.Slice.AsRaw` functions (#5298)
```

Comment:
```suggestion
- `pdata`: Expose `pcommon.NewSliceFromRaw` and `pcommon.Slice.AsRaw` functions (#5298)
```

Reply:
Also check the PR# here, I don't believe this is correct.

Reply:
Thanks, done.